brianlong/www.validators.app
# 186700974 add ping failiures to stats

#### What's this PR do?
- add number of failiures to ping thing recent stats table

#### How should this be manually tested?
- run migrations
- run tests
- add some ping things: `rails r dev/create_test_ping_things.rb`
- run worker: `rails r "PingThingRecentStatsWorker.perform_async('mainnet')"`
- check if data in the table matches the created ping things
- check design for multiple screen resolutions

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186700974)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
